modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Review/index.tsx

```diff
@@ -0,0 +1,115 @@
+/* eslint-disable sort-keys */
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayTable from '@clayui/table';
+import React, {useContext} from 'react';
+
+import {NewApplicationAutoContext} from '../../../context/NewApplicationAutoContextProvider';
+const Review = () => {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [state] = useContext(NewApplicationAutoContext);
+  const data = [
+    {
+      COVERAGE: 'BODILY INJURY',
+      PROTECTIONAMOUNT: '$0.00 / $0.00',
+      PREMIUM: '$0.00 ',
+    },
```

Comment:
It's a good practice to write the object keys in lowercase :)

```suggestion
    {
      coverage: 'BODILY INJURY',
      protectionAmount: '$0.00 / $0.00',
      premium: '$0.00 ',
    },
```

Reply:
thanks